src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -79,6 +77,37 @@
         this.ec2SGs = ec2SGbuilder.build();
 
         /*
+         * IAM keys
+         */
+
+        log.info("Getting IAM keys");
+        String accountNumber = ""; // we need this to describe RDS
+        final ImmutableList.Builder<IAMUserWithKeys> usersBuilder = new ImmutableList.Builder<IAMUserWithKeys>();
+
+        final ListUsersRequest listUsersRequest = new ListUsersRequest();
+        ListUsersResult listUsersResult;
+        do {
+            log.debug("Performing IAM request: {}", listUsersRequest);
+            listUsersResult = iamClient.listUsers(listUsersRequest);
+            final List<User> users = listUsersResult.getUsers();
+            log.debug("Found {} users", users.size());
+            for (User user : users) {
```

Comment:
I'm OK with adding an optional `accountNumber` arg. But in the rare case when `accountNumber` is absent and the aws account has no user, billow will still fail to describe RDS.


An interesting thing is that when given someone else's accountNumber and the correct key/secret credentials, billow will be able to describe resources in other accounts 🙂